include/xen/be/XenGnttab.hpp

```diff
@@ -161,7 +161,8 @@ class XenGnttabDmaBufferExporter
 	 *            will be produced
 	 * @param[in] refs  vector of grant reference ids
 	 */
-	XenGnttabDmaBufferExporter(domid_t domId, const GrantRefs &refs);
+	XenGnttabDmaBufferExporter(domid_t domId, const GrantRefs &refs,
+							   const size_t offset = 0);
```

Comment:
No need for const for passing int type parameter since it is copied and passed value is not modified any case.

Reply:
fixed